website/content/commands/index.mdx

```diff
@@ -55,26 +55,58 @@ For example, to see help about the up subcommand:
 $ waypoint up -h
 Usage: waypoint up [options]
 
-  Perform the build, deploy, and release steps for the project.
```

Comment:
I'd vote for leaving `for the project` here - it answers the question "what will waypoint build, deploy, and release?"

Reply:
@izaaklauer Sorry for the confusion! This is actually taken from the actual CLI output of `$ waypoint up -h` with the latest version, so I leave this section as-is...!